audit/projects/k8s-staging-sig-storage/buckets/artifacts.k8s-staging-sig-storage.appspot.com/metadata.txt

```diff
@@ -11,8 +11,8 @@ gs://artifacts.k8s-staging-sig-storage.appspot.com/ :
 	Labels:				None
 	Default KMS key:		None
 	Time created:			Wed, 10 Jun 2020 03:16:58 GMT
-	Time updated:			Thu, 09 Sep 2021 16:49:19 GMT
-	Metageneration:			11
+	Time updated:			Fri, 24 Sep 2021 21:30:53 GMT
+	Metageneration:			12
```

Comment:
These are all from running `ensure-staging-storage.sh` for all projects as part of https://github.com/kubernetes/k8s.io/pull/2813

For some reason the bucket logging stuff still isn't idempotent